upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -380,3 +383,22 @@ func (tf *TemplateFunctions) KopsSystemEnv() []corev1.EnvVar {
 
 	return envMap.ToEnvVars()
 }
+
+// OpenStackCCM returns OpenStack external cloud controller manager current image
+// with tag specified to k8s version
+func (tf *TemplateFunctions) OpenStackCCM() string {
+	var tag string
+	parsed, err := util.ParseKubernetesVersion(tf.cluster.Spec.KubernetesVersion)
+	if err != nil {
+		tag = "latest"
+	} else {
+		if parsed.Minor == 13 {
+			// known bug release
+			tag = "v1.13.1"
```

Comment:
The bugfix release, and also the .0 version of `1.13` does not start with a `v`

Reply:
hopefully from now on they will use v in name... I will fix this